content/sklearn/concepts/clustering/clustering.md

```diff
@@ -0,0 +1,119 @@
+---
+Title: 'Clustering'
+Description: 'Grouping similar data points together based on defined characteristics or features.'
+Subjects:
+  - 'Data Science'
+  - 'Machine Learning'
+Tags:
+  - 'Clustering'
+  - 'Machine Learning'
+  - 'Scikit-learn'
+  - 'Unsupervised learning'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+**Clustering** is an unsupervised machine learning technique that groups similar rows of unlabeled data. Various clustering algorithms (such as k-means, DBSCAN etc.) apply different type of clustering. 
+
+Clustering is primarily used for an exploratory data analysis. Each technique varies in understanding what constitutes a cluster and how to find them so that different algorithms may yield different clusters for the same dataset. Since there is no ground truth to verify against, it is essential to evaluate the results. Iterating to find satisfactory clusters is vital until the output quality meets expectations. 
```

Comment:
```suggestion
Clustering is primarily used for exploratory data analysis. Each technique varies in understanding what constitutes a cluster and how to find them so that different algorithms may yield different clusters for the same dataset. Since there is no ground truth to verify against, it is essential to evaluate the results. Iterating to find satisfactory clusters is vital until the output quality meets expectations. 
```